src/main/java/uk/gov/justice/hmpps/prison/api/resource/ServiceAgencySwitchesResource.kt

```diff
@@ -38,4 +41,26 @@ class ServiceAgencySwitchesResource(private val service: ServiceAgencySwitchesSe
     serviceCode: String,
   ): List<PrisonDetails> =
     service.getServicePrisons(serviceCode)
+
+  @ApiResponses(
+    ApiResponse(responseCode = "200", description = "OK"),
+    ApiResponse(responseCode = "401", description = "A valid auth token was not presented", content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))]),
```

Comment:
no response for the `400` scenario where the service / prison combination already exists

Reply:
Yes - though probably better as a 409